src/integrationTest/java/uk/gov/hmcts/darts/dailylist/service/DailyListProcessorTest.java

```diff
@@ -67,14 +72,18 @@ void dailyListProcessorMultipleDailyList() throws IOException {
         CourthouseEntity swanseaCourtEntity = dartsDatabase.createCourthouseWithTwoCourtrooms();
         LocalTime dailyListTIme = LocalTime.of(13, 0);
         DailyListEntity dailyListEntity = DailyListTestData.createDailyList(dailyListTIme, String.valueOf(SourceType.CPP),
-                swanseaCourtEntity, "tests/dailyListProcessorTest/dailyListCPP.json");
+                                                                            swanseaCourtEntity, "tests/dailyListProcessorTest/dailyListCPP.json"
+        );
 
         DailyListEntity oldDailyListEntity = DailyListTestData.createDailyList(dailyListTIme.minusHours(3),
-                String.valueOf(SourceType.CPP), swanseaCourtEntity, "tests/dailyListProcessorTest/dailyListCPP.json");
+                                                             String.valueOf(SourceType.CPP),
+                                                             swanseaCourtEntity,
```

Comment:
I assume this will no longer work because the courthouse entity is not linked to the daily list entity?